<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(186, 189, 182);</string>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Döviz/Altın</string>
       </attribute>
       <widget class="QWidget" name="widget_9" native="true">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>261</width>
          <height>431</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QWidget" name="widget" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Dolar: </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_usd">
              <property name="text">
               <string>Dolar</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_2" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Euro: </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_eur">
              <property name="text">
               <string>Euro</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_3" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Altın: </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_gold">
              <property name="text">
               <string>Altın</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_4" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Sterlin:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_sterlin">
              <property name="text">
               <string>STERLIN</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_5" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>BIST 100:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_bist">
              <property name="text">
               <string>BIST 100</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_6" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>Bitcoin:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_bit">
              <property name="text">
               <string>BITCOIN</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_7" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>Gram Gümüş:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_silver">
              <property name="text">
               <string>GRAM GÜMÜŞ</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_8" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>Brent:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_brent">
              <property name="text">
               <string>BRENT</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>20</y>
          <width>491</width>
          <height>471</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../../../Downloads/ChatGPT Image 21 Ara 2025 14_35_11.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>20</y>
          <width>161</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Güncel Kurlar</string>
        </property>
       </widget>
       <zorder>label_11</zorder>
       <zorder>widget_9</zorder>
       <zorder>label_12</zorder>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Dönüştürme</string>
       </attribute>
       <widget class="QWidget" name="widget_14" native="true">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>60</y>
          <width>561</width>
          <height>381</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QWidget" name="widget_10" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_9">
            <item>
             <widget class="QLabel" name="label_14">
              <property name="text">
               <string>Dönüştürmek istediğiniz miktar:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="input_amount">
              <property name="styleSheet">
               <string notr="true">QLineEdit {
    border: 1px solid #ccc;
    border-radius: 5px;
    padding: 5px;
    background-color: #f9f9f9;
}</string>
              </property>
              <property name="inputMask">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_11" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_10">
            <item>
             <widget class="QLabel" name="label_15">
              <property name="text">
               <string>Dönüştürmek istediğiniz döviz türü:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="combo_from">
              <item>
               <property name="text">
                <string>GRAM ALTIN</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>BRENT</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>DOLAR</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>EURO</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>BİTCOİN</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>STERLİN</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>BİST 100</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>GRAM GÜMÜŞ</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_12" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_11">
            <item>
             <widget class="QLabel" name="label_17">
              <property name="text">
               <string>Dönüştürmek istediğiniz para cinsi: </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="combo_to">
              <item>
               <property name="text">
                <string>TL</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_convert">
           <property name="text">
            <string>Hesapla</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_13" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_12">
            <item>
             <widget class="QLabel" name="label_16">
              <property name="text">
               <string>Değer: </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_result">
              <property name="text">
               <string>Değer</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QLabel" name="label_18">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>30</y>
          <width>251</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Döviz Çevirici</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Grafik</string>
       </attribute>
       <widget class="QWidget" name="graphWidget" native="true">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>50</y>
          <width>691</width>
          <height>431</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_20">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>20</y>
          <width>261</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>GRAM ALTIN SON DEĞİŞİMLERİ </string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Yatırımlarım</string>
       </attribute>
       <widget class="QTableWidget" name="table_investments">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>220</y>
          <width>511</width>
          <height>251</height>
         </rect>
        </property>
        <column>
         <property name="text">
          <string>Varlık</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Miktar</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Fiyat</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Toplam</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Tarih</string>
         </property>
        </column>
       </widget>
       <widget class="QWidget" name="widget_17" native="true">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>30</y>
          <width>361</width>
          <height>141</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QWidget" name="widget_15" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_13">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Yatırım Türü Seçin: </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="combo_asset">
              <item>
               <property name="text">
                <string>GRAM ALTIN</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>DOLAR</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>EURO</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>GRAM GÜMÜŞ</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>BITCOIN</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_16" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_14">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Yatırım Mikatarı Girin: </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="input_amount_invest">
              <property name="styleSheet">
               <string notr="true">QLineEdit {
    border: 1px solid #ccc;
    border-radius: 5px;
    padding: 5px;
    background-color: #f9f9f9;
}</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_add_investment">
           <property name="text">
            <string>Yatırım Ekle</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QLabel" name="label_19">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>190</y>
          <width>231</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Yaptığınız Tüm Yatırımlar:</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>Profilim</string>
       </attribute>
       <widget class="QWidget" name="widget_guest" native="true">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>60</y>
          <width>521</width>
          <height>291</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="lbl_guest_info">
           <property name="text">
            <string>Giriş Yapmadınız</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>Yatırımlarım Sekmesine Girmek İçin Giriş Yapmanız Gerekmektedir.</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_login">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(78, 154, 6);</string>
           </property>
           <property name="text">
            <string>Giriş Yap</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_register">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(196, 160, 0);</string>
           </property>
           <property name="text">
            <string>Kayıt Ol</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="widget_user" native="true">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>70</y>
          <width>531</width>
          <height>311</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="lbl_username">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Hoş Geldiniz: Kullanıcı</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_change_password">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(78, 154, 6);</string>
           </property>
           <property name="text">
            <string>Şifre Değiştir</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_change_account">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(32, 74, 135);</string>
           </property>
           <property name="text">
            <string>Hesap Değiştir</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_logout">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(164, 0, 0);</string>
           </property>
           <property name="text">
            <string>Çıkış Yap</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <zorder>widget_user</zorder>
       <zorder>widget_guest</zorder>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
